Library/Formula/cairo.rb

```diff
@@ -8,11 +8,9 @@ class Cairo < Formula
   keg_only :provided_pre_mountain_lion
 
   option :universal
-  # Tiger's X11 is simply way too old
-  option 'without-x', 'Build without X11 support' if MacOS.version > :tiger
 
   depends_on "pkg-config" => :build
-  depends_on :x11 => :optional if MacOS.version > :leopard
+  depends_on :x11 => if MacOS.version > :leopard then :optional else :recommended end
```

Comment:
I'd prefer to see this put into multiple lines for readability:

```ruby
if MacOS.version > :leopard
  depends_on :x11 => :optional
else
  depends_on :x11 => :recommended
end
```

Incidentally, why is this recommended on Tiger?